docs/src/components/ComponentDoc/ComponentDoc.js

```diff
@@ -73,12 +73,14 @@ class ComponentDoc extends Component {
                 subheader={_.join(componentInfo.docblock.description, ' ')}
               />
               <ComponentDocSee seeTags={seeTags} />
-              <ComponentDocLinks
-                displayName={displayName}
-                parentDisplayName={componentInfo.parentDisplayName}
-                repoPath={componentInfo.repoPath}
-                type={componentInfo.type}
-              />
+              {componentInfo.repoPath && (
```

Comment:
Hides this block as code is not hosted in this repo:

![image](https://user-images.githubusercontent.com/14183168/90385990-2fc58480-e084-11ea-9d4b-48f07bc0b992.png)